contracts/BattleController.sol

```diff
@@ -199,13 +200,18 @@ contract BattleController is Upgradable {
         uint256 _hatchingPrice,
         uint256 _dragonsAmount
     ) internal pure returns (uint256) {
-        uint8 _factor;
-
-        if (_dragonsAmount < 15000) _factor = 20;
-        else if (_dragonsAmount < 30000) _factor = 10;
-        else _factor = 5;
-
-        return _hatchingPrice.mul(_factor).div(PERCENT_MULTIPLIER);
+        uint32 _factor;
+
+        if (_dragonsAmount < 3000) _factor = 2000000; // Hydrogen Eon
```

Comment:
`MAX_GOLD_REWARD_MULTIPLIER` is less than the `_factor` value. In this case we will pay ```1000 * 2000000 / 100000 =  20000``` gold to the winner